components/expenses/PayoutMethodSelect.js

```diff
@@ -239,16 +258,58 @@ class PayoutMethodSelect extends React.Component {
   });
 
   render() {
-    const { payoutMethods, defaultPayoutMethod, payoutMethod, ...props } = this.props;
+    const { payoutMethods, defaultPayoutMethod, payoutMethod, payee, collective, ...props } = this.props;
+
     const { removingPayoutMethod } = this.state;
     const value = !isEmpty(payoutMethod) && this.getOptionFromPayoutMethod(payoutMethod);
+
+    const payeeIsCollectiveFamilyType = payee && AccountTypesWithHost.includes(payee.type);
+    const payeeIsSameHost = payee && payee.host?.id === collective.host?.id;
+
+    const styledSelectOptions = this.getOptions(payoutMethods, payee);
+    const hasSuitablePayoutMethodOption = styledSelectOptions.find(({ options }) => options.length > 0) ? true : false;
+
+    if (payeeIsCollectiveFamilyType && !payeeIsSameHost) {
+      if (!collective?.host?.isTrustedHost) {
+        return (
+          <MessageBox type="error" mt={2} mb={3} fontSize="12px">
+            <FormattedMessage defaultMessage="This Expense is between different Hosts but the Payer Host is not allowed for this yet." />
+            &nbsp;
+            <FormattedMessage defaultMessage="If it's an issue, contact the Host or Open Collective support." />
```

Comment:
Why not this:

```suggestion
            <FormattedMessage defaultMessage="This Expense is between different Hosts but the Payer Host is not allowed for this yet. If it's an issue, contact the Host or Open Collective support." />
```

Reply:
The goal was to reuse the second part elsewhere.